.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to igraph
+
+This outlines how to propose a change to igraph.
+
+## Fixing typos
+
+You can fix typos, spelling mistakes, or grammatical errors in the documentation directly using the GitHub web interface, as long as the changes are made in the _source_ file. 
+This generally means you'll need to edit [roxygen2 comments](https://roxygen2.r-lib.org/articles/roxygen2.html) in an `.R`, not a `.Rd` file. 
+You can find the `.R` file that generates the `.Rd` by reading the comment in the first line.
```

Comment:
Will people need to regenerate the `.Rd` file themselves (how?), or will this be done for them?

Reply:
we'd do it for them for now